src/components/panels/SortiePanel.vue

```diff
@@ -16,31 +16,23 @@
         :key="`sortie-${index}`"
         v-bind:class="{
           'list-group-item-borderless': index !== sortie.variants.length - 1,
+          'pb-0': index !== sortie.variants.length - 1,
           'list-group-item-borderbottom': index === sortie.variants.length - 1,
         }"
       >
-        <div class="indent-1">
-          <span class="pull-left">
+        <div class="ml-3">
+          <span class="pull-left pr-2">
             <b>{{ mission.missionType }} - {{ mission.node }}</b>
           </span>
-          <br />
-          <span v-b-tooltip.right :title="mission.modifierDescription" class="pull-left">
+          <span v-b-tooltip.top :title="mission.modifierDescription" class="pull-left">
```

Comment:
could we pull this right since we're not breaking between lines?
```suggestion
          <span v-b-tooltip.top :title="mission.modifierDescription" class="pull-right">
```